index.html

```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
         </div>
       </div>
     </section>
+    <!-- Contact Form -->
+    <footer>
+      <div class="contact-form"></div>
+      <h2 class="contact-me">Contact me</h2>
+      <p class="para-text">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mbjwvgbk" id="comment">
```

Comment:
- [ ] Great job adding the `action` attribute 👍 . But, like the previous reviewer mentioned you need to also add the `method` attribute. Without the `method` attribute we get this error when submitting the form 🙏 .

![image](https://user-images.githubusercontent.com/74814780/177587576-1fe5df1b-abba-4b70-aca0-6cc4f7f359d3.png)

Reply:
This issue is well resolved! ✔️